specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md

```diff
@@ -10,12 +10,13 @@ Environment variables is one example, where there are several existing APIs
 that provide partially overlapping functionality, and with some gaps. This spec
 describes the Reunion support for environment variables.
 
-Important Note: this feature relies on other platform features which are only
-available from Windows summer 2021 onwards. Therefore this feature is also 
+Important Note: The EV tracking feature relies on other platform features which are only
+available from Windows summer 2021 onwards. Therefore EV tracking is also 
```

Comment:
>Windows summer 2021 onwards

You mean 21H1? 

SUGGESTION: `...Windows 21H1+ (i.e. version >=10.0.19043.0).`

P.S. I find the table [here](https://en.wikipedia.org/wiki/Windows_10_version_history#Channels) very handy as I can never remember which version has what name / codename / build / date / marketing label.